spec/rental_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../rental'
+
+describe Rental do
+  let(:book) { Book.new('Animal Farm', 'George Orwell') }
+  let(:person) { Person.new(25, 'John') }
+  subject { Rental.new('2023-09-13', book, person) }
+
+  it 'should have a date' do
+    expect(subject.date).to eq('2023-09-13')
+  end
```

Comment:
Well! your current test cases in `rental_spec.rb` are working fine. But it's important to add a test case for the `to_h` method in the `rental_spec.rb` file. Because as per [project requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md#:~:text=Add%20unit%20tests%20for%20all%20the%20methods%20in%20all%20the%20classes%3A) you should add test cases for all methods in all classes. Please kindly add a test case for the `to_h` method which is inside the `rental.rb` file.